CMSgov/eAPD
# tforkner/3556-move-budget moved budget to common

Resolves #3556

### Description
Moving the budget information from the frontend to the shared package

### Significant changes or possible side effects
Possible budget issues

### Steps to manually verify this change
Make sure that nothing seems weird with the budget. I feel like there are enough budget tests that we can probably mostly rely on them to verify this change.

### This pull request is ready to review when


### This pull request can be merged when
